packages/extras/src/lib/types/components.ts

```diff
@@ -1,99 +1,101 @@
 import type {
```

Comment:
I really don't know why VScode does this!!!

Reply:
Is this encoding issue?

Reply:
Yeah, I think it was indented with spaces before, so this is fine!